src/edgekv/ekv-handler.ts

```diff
@@ -4,30 +4,32 @@ import * as response from './ekv-response';
 import * as ekvhelper from './ekv-helper';
 import * as edgeWorkersSvc from '../edgeworkers/ew-service';
 
-export async function listNameSpaces(environment: string, details) {
+export async function listNameSpaces(environment: string, details: boolean, sortDirection: cliUtils.sortDirections, orderBy: string) {
   ekvhelper.validateNetwork(environment);
-  let nameSpaceList = await cliUtils.spinner(
+  const nameSpaceList = await cliUtils.spinner(
     edgekvSvc.getNameSpaceList(environment, details),
     'Fetching namespace list...'
   );
   if (nameSpaceList != undefined && !nameSpaceList.isError) {
-    let nsListResp = [];
+    const nsListResp = [];
     if (nameSpaceList.hasOwnProperty('namespaces')) {
-      let namespace = nameSpaceList['namespaces'];
+      const namespace = nameSpaceList['namespaces'];
+      cliUtils.sortObjectArray(namespace, orderBy, sortDirection);
+
       namespace.forEach(function (value) {
         if (details) {
-          let retentionPeriod = ekvhelper.convertRetentionPeriod(
+          const retentionPeriod = ekvhelper.convertRetentionPeriod(
             value['retentionInSeconds']
           );
-          let groupId = value['groupId'] == undefined ? 0 : value['groupId'];
+          const groupId = value['groupId'] == undefined ? 0 : value['groupId'];
           nsListResp.push({
-            Namespace: value['namespace'],
+            NamespaceId: value['namespace'],
             RetentionPeriod: retentionPeriod,
             GeoLocation: value['geoLocation'],
-            'Access GroupId': groupId,
```

Comment:
changed this to be 1 word so it's consistent with our other headers and is also less confusing for users to try and order by

Reply:
@mryakan  Just want to clarify what exactly we should be renaming here? I believe the object key is already called 'AccessGroupId' on [line 29](https://github.com/akamai/cli-edgeworkers/pull/98/files#diff-ad52197919319175e068cf2fafbfb2fe072d14714ba32e813fb7adaad9b682f9R29) (all I did was remove the space). Or are you referring the the variable name for the value on [line 24](https://github.com/akamai/cli-edgeworkers/pull/98/files#diff-ad52197919319175e068cf2fafbfb2fe072d14714ba32e813fb7adaad9b682f9R24)?

Reply:
I want to make sure 2 things:
- The column heading in the cli output has "Access" in the title not just groupId
- The order-by option name is the same as the column title
I may have misread this, but I assumed this meant the column header was being renamed to just "groupId".

Reply:
No worries, groupId is just the variable name for the object value. The object key name and column name is `AccessGroupId` and can be sorted using that name with any combination of capitals and lowercase.

![image](https://user-images.githubusercontent.com/33794710/192365584-cebec753-cf98-4f46-bd1f-b9510c4958be.png)

Reply:
ok, we're all set then. Thanks for the clarification.